src/dog.sol

```diff
@@ -105,6 +105,10 @@ contract Dog is LibNote {
     function mul(uint256 x, uint256 y) internal pure returns (uint256 z) {
         require(y == 0 || (z = x * y) / y == x);
     }
+    function div(uint256 x, uint256 y) internal pure returns (uint256 z) {
```

Comment:
This can be removed now.
Btw why would we need this function? Dividing by zero already throws by default (at least in solidity, not 100% sure at EVM level).
I think older versions of solidity didn't catch it, but should have been solved long time ago.